src/components/Wallet.tsx

```diff
@@ -34,11 +42,18 @@ const buttons = [
     icon: <NavigationPointerIcon />,
     route: "/transfer"
   }
+] as const;
+
+const placeHolder = [
```

Comment:
Placeholder is a single word

```suggestion
const placeholder = [
```